frontend/internal-packages/pglite-server/src/extensions/pgtap/index.ts

```diff
@@ -1,8 +1,32 @@
+import { existsSync } from 'node:fs'
+import { join } from 'node:path'
+import { pathToFileURL } from 'node:url'
 import type { Extension } from '@electric-sql/pglite'
 
 export const pgtap: Extension = {
   name: 'pgtap',
-  setup: async () => ({
-    bundlePath: new URL('./pgtap.tar.gz', import.meta.url),
-  }),
+  setup: async () => {
+    // In Next.js server environment, check for webpack-generated file
+    if (process?.cwd) {
+      // Next.js puts webpack assets in .next/server/static/extensions/
+      const nextServerPath = join(
+        process.cwd(),
+        '.next',
+        'server',
+        'static',
+        'extensions',
+        'pgtap.tar.gz',
+      )
+      if (existsSync(nextServerPath)) {
+        return {
+          bundlePath: pathToFileURL(nextServerPath),
+        }
+      }
+    }
+
+    // Fallback to original relative path for non-Next.js environments (tests, etc.)
+    return {
+      bundlePath: new URL('./pgtap.tar.gz', import.meta.url),
+    }
+  },
```

Comment:
Thanks for fixing it!
FYI: I was planning to submit a PR for this extension to the PGLite repository later. Since zaza-san actually got it working this time, I'll submit the PR soon!